pithos/plugins/_mpris.py

```diff
@@ -144,6 +128,40 @@ def _update_metadata(self, song):
         return self._metadata
 
     @property
+    def _default_cover_url(self):
+        """
+        Many MPRIS widgets/applets have trouble with svg icons. They load them at their resolution
+        and then scale them leading to blurry default covers in the widget/applet.
+        Also some icon themes are not scalable and contain no svg icons so
+        Gtk.IconTheme.get_default().lookup_icon('audio-x-generic',-1, 0) does not return an svg but a low res png.
+        This will load, scale and cache a default cover as best as we can to avoid poor quality default covers.
+        """ 
+        def lookup(current_theme):
+            common_icon_sizes = [512, 256, 128, 96, 64, 48, 24, 16, -1]
+            for i in common_icon_sizes:
+                icon_lookup = current_theme.lookup_icon('audio-x-generic', i, 0)
+                if icon_lookup is not None:
+                    break
+            return icon_lookup.get_filename()
+        try:
+            if self.window.tempdir is not None:
+                current_icon_theme_name = Gtk.Settings.get_default().props.gtk_icon_theme_name
+                generic_cover_path = self.window.tempdir.name + '/default-cover-' + current_icon_theme_name
+                generic_cover = Gio.file_new_for_path(generic_cover_path)
+                if not generic_cover.query_exists():
+                    icon_path = lookup(Gtk.IconTheme.get_default())
+                    icon_pixbuf = GdkPixbuf.Pixbuf.new_from_file_at_scale(icon_path, 500, -1, True)
```

Comment:
That's the size of a cover that comes from Pandora. I wanted them to be the same size. I don't want the MPRIS applet/widget to handle the scaling. Letting them handle upscaling is what leads to crappy image quality. Many rasterize svg's at there exact file size and then scale them lossely up.